keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -214,6 +230,8 @@ For alpha, exhaustive Kubelet unit tests will be provided. Functions affected by
 ```
 [TestShouldPullImage link](https://github.com/kubernetes/kubernetes/pull/94899/files#diff-7297f08c72da9bf6479e80c03b45e24ea92ccb11c0031549e51b51f88a91f813R311-R438)
 
+PersistHashMeta()
+
 At beta we should revisit if integration buckets are warranted for e2e node and/or cri-tools/critest, and after gathering feedback.
 
 ### Graduation Criteria
```

Comment:
For PRR, I would recommend changing all the `N/A` to something like "TBD" as they are all applicable, just not required for alpha.  Also, you will definitely need metrics or similar for this in beta.

> ###### How can an operator determine if the feature is in use by workloads?

The current answer to this question is not acceptable as-is (it is not a realistic way for admins to understand that this feature is active).  You need metrics or similar.

Reply:
should be resolved..